dlt/sources/.gitignore

```diff
@@ -0,0 +1,10 @@
+# ignore secrets, virtual environments and typical python compilation artifacts
```

Comment:
I'm not sure this will be distributed with the pypi package? should we place all non-importable code, templates and files in some kind of "data" folder?